mmcv/ops/roi_align.py

```diff
@@ -94,7 +94,7 @@ def forward(ctx,
     def backward(ctx, grad_output):
         rois, argmax_y, argmax_x = ctx.saved_tensors
         grad_input = grad_output.new_zeros(ctx.input_shape)
-
+        grad_output = grad_output.contiguous()
```

Comment:
Add comments above this line in the code to indicate that in some cases the `grad_output` is incontiguous thus this modification is necessary.